WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -152,7 +159,16 @@ private extension OrdersViewController {
         resultsController.predicate = {
             let excludeSearchCache = NSPredicate(format: "exclusiveForSearch = false")
             let excludeNonMatchingStatus = statusFilter.map { NSPredicate(format: "status = %@", $0.rawValue) }
-            let predicates = [ excludeSearchCache, excludeNonMatchingStatus ].compactMap { $0 }
+
+            let futureDate = getTomorrow()
+            var predicates: [NSPredicate]
+            
+            if let tomorrow = futureDate {
+                let dateSubPredicate = NSPredicate(format: "dateCreated < %@", tomorrow as NSDate)
+                predicates = [ excludeSearchCache, excludeNonMatchingStatus, dateSubPredicate ].compactMap { $0 }
```

Comment:
Style nit: You could probably tighten up this code with something like:

```
...
var predicates = [ excludeSearchCache, excludeNonMatchingStatus ].compactMap { $0 }
if let tomorrow = getTomorrow() {
    let dateSubPredicate = NSPredicate(format: "dateCreated < %@", tomorrow as NSDate)
    predicates.append(dateSubPredicate)
}
...
```

Reply:
thank you! That `if/else` was really bothering me.